config/action.d/firewallcmd-common.conf

```diff
@@ -5,72 +5,102 @@
 
 [Init]
 
+# Option:  bantime
+# Notes:   Specifies the bantime in seconds (handled internally rather than by fail2ban)
+# Values:  [ NUM ]  Default: 600
+
+bantime = 600
```

Comment:
Imho, the `bantime` is only required for the actions, that are not working without it (expected in cmd-line), resp. has no unban-functionality at all (self-unbanning).
For many of these a default timeout (0 or -1) for infinite can be set, so will be unbanned by remove rule during call of unban (except these, that are not have it).
The passing of a bantime is configurable even as `bantime` parameter in brackets by defining of an action in jail.conf.
Why and whether it will be expected in firewallcmd I don't know (I don't use it at all)...

We will get a problem with such actions, as long as we will integrate ban-time-incr branch. Because these actions should be treated differently. 
Roughly described flow that currently implemented in ban-time-incr:
- if failure occurred, it will be added to failmanager immediately, that send an event to observer
- asynchronous observer checks the database, to determine IP would be bad, if yes observer "decreases" `maxretry` for this IP;
- ban occurred immediately if reached `maxretry`;
- asynchronous observer checks the database, to determine IP would be bad;
- if bad, observer prolongs a time of ban in banmanager only (without calling any action again)
- ...
- banmanager calls unban if the ban time was reached